pkg/web/routes.go

```diff
@@ -169,10 +169,15 @@ func (s *Server) setupRoutes() (err error) {
 			transactions.GET("/export", authorize(permiss.TravelRuleManage), s.ExportTransactions)
 
 			// Transaction specific actions
+			// TODO: deprecate /:id/preview
 			transactions.POST("/:id/send", authorize(permiss.TravelRuleManage), s.SendEnvelopeForTransaction)
-			transactions.GET("/:id/preview", authorize(permiss.TravelRuleManage), s.AcceptTransactionPreview)
+			transactions.GET("/:id/payload", authorize(permiss.TravelRuleView), s.LatestPayload)
+			transactions.GET("/:id/preview", authorize(permiss.TravelRuleView), s.AcceptTransactionPreview)
+			transactions.GET("/:id/accept", authorize(permiss.TravelRuleView), s.AcceptTransactionPreview)
```

Comment:
I would like to replace the `GET /:id/preview` endpoint with `GET /:id/accept` -- lmk what you think.

Reply:
To maintain consistency, I think this change will be fine.

Reply:
When you switch to the new endpoint, would you mind deleting the old endpoint?

Reply:
Sure, I can do that.